assets/dev/js/editor/elements/widgets/base.js

```diff
@@ -0,0 +1,14 @@
+import ElementBase from 'elementor-elements/elements/base';
+
+export default class WidgetBase extends ElementBase {
+	/**
+	 * @return {string}
+	 */
+	getWidgetType() {
+		elementorModules.ForceMethodImplementation();
+	}
+
+	getTypeKey() {
+		return this.getType() + '-' + this.getWidgetType();
```

Comment:
We need to merge all elements in one format. e.g. `element-section`, `element-column`, `widget-heading`

Reply:
Handled.

Reply:
Handled.